packages/editor/src/components/widgets/input/InputWithBrowser.tsx

```diff
@@ -0,0 +1,27 @@
+import './Input.css';
+import { Browser, BrowserType, useBrowser } from '../../../components/browser';
+import { usePath } from '../../../context';
+import { ComponentProps, forwardRef } from 'react';
+import Input from './Input';
+import { typeFilter } from '../../../components/browser/Browser';
+
+export type InputProps = Omit<ComponentProps<'input'>, 'value' | 'onChange' | 'ref'> & {
```

Comment:
Better naming -> InputWithBrowserProps
And I think you don't use this type somewhere else, so you don't have to export it